cernopendata_client/searcher.py

```diff
@@ -203,8 +203,17 @@ def get_files_list(
         files_list_expanded = []
         for file_ in files_list:
             if file_[0].endswith("_file_index.json"):
-                url_file = file_[0].replace(SERVER_ROOT_URI, server)
-                json_files = requests.get(url_file).json()
+                try:
+                    url_file = "{}/record/{}/files/{}".format(
```

Comment:
This fixes the problem at hand.

That said, I'm wondering whether we shouldn't introduce at this point in time already the distinction between EOSPUBLIC locations and `/record/NNN/files/foo.root` locations. Since this topic will touch other parts, I'll probably make another ticket about this.